### ISSUE TITLE
Handle multiple accounts with the same email address in multiple directories

### ISSUE THREAD

**Issue by mdeggies** [okta/stormpath-migration#38]
Use Case: In Stormpath, I have dev/staging/prod resources within a single Stormpath tenant. In some cases, I have accounts with the same email address in different directories. 

Goal: I want to keep all of my dev accounts (and other dev resources) within one Okta org, all of my staging accounts (and all other staging resources) within another Okta org, etc. 

Problem: There are only 2 options right now for dealing with accounts with the same email address: 

1. Link the accounts with the same email addresses together, and then export/import. This means we want to merge dev/staging/prod accounts into one Okta account. This does not meet my needs.

2. Don't link accounts with the same email addresses, and then export/import. This means a single account will be created in Okta, based on whichever account came first (ordered by the ID of the application/org mappings). This does not meet my needs. In many cases, we see a seemingly random-ish mix of dev/staging/prod accounts created, and I don't have control over which type of accounts are created -- e.g. I cannot just say 'Import only my dev accounts!' 

Current Workarounds:  

1. Troll through the export data, and delete the account files that we don't want to import.

2. Use [the Stormpath migration tool](https://github.com/stormpath/stormpath-migrate) to copy the contents your tenant into three different tenants (that already exist, because SP registration has been disabled). One tenant will be dedicated to only dev resources, so delete all staging/prod resources. Another tenant will be dedicated to only staging resources, so delete all dev/prod resources. Etc...

**Comment by mdeggies** [comment]
@robertjd's suggestion: Add an option to import accounts by only following account store mappings (rather than trying to import all accounts from the tenant).

**Comment by tirthb** [comment]
It's urgent for us.

**Comment by mdeggies** [comment]
@tirthb, if I recall correctly, we helped your team with workaround #2. How has that been working for you? If you need any more assistance, you're welcome to reach back out at developers@okta.com or support@stormptah.com.

**Comment by tirthb** [comment]
Yes, it met our needs. Thanks for helping out in the migration using our old tenants.
